server.conf.in

```diff
@@ -127,6 +127,25 @@
 # Defaults to firefox
 # browser = firefox
 
+# Path to the Selenium driver to use for recordings.
+# If set the driver must match the browser being used (for example,
+# "/usr/bin/geckodriver" for "firefox"). If no driver is explicitly set Selenium
+# Manager will try to find the right one in $PATH, downloading it as a fallback.
+# Note that Selenium Manager does not work in some architectures (for example,
+# Linux on arm64/aarch64), so in those architectures the driver must be
+# explicitly set.
+#driver =
```

Comment:
After thinking a bit more about this I think it would be better to name the option `driverPath` rather than just `driver`, as `driver` seems to imply just the driver name (similar to `browser`).